client/resp.go

```diff
@@ -406,20 +406,22 @@ func (c *Conn) readResultRows(result *Result, isBinary bool) (err error) {
 
 		result.RowDatas = append(result.RowDatas, data)
 	}
-
+	/*
```

Comment:
let's undo this change for now. we should make a builtin to fill in `results.Values` and call it everytime.